include/npc.h

```diff
@@ -98,7 +98,7 @@ typedef struct NpcSettings {
     /* 0x20 */ s32 flags;
     /* 0x24 */ char unk_24[4];
     /* 0x28 */ s16 level;
-    /* 0x2A */ s16 unk_2A;
+    /* 0x2A */ s16 unk_2A;  // action flags: 1 = jump on seeing player
```

Comment:
Should rename this field & make a flags enum for it

Reply:
Not enough is known about this field to give it a definitive name. An enum has already been created for the values it may hold.